eftest/src/eftest/runner.clj

```diff
@@ -182,7 +191,10 @@
        (binding [report/*context* (atom {})
                  test/report      (:report opts progress/report)]
          (test/do-report {:type :begin-test-run, :count (count vars)})
-         (let [counters (test-all vars opts)
+         (let [executor (threadpool-executor opts)
+               opts′    (assoc opts :executor executor)
```

Comment:
That tick at the end of `opts′` doesn't look like it's ASCII. I don't think you'd need a new variable name here either; just use `opts`.

Reply:
It isn’t! I didn’t know you preferred ASCII-only; fixing this now. (I personally like that Clojure supports “real” characters and I like to use them in my own code, but I understand that’s not for everyone!) As for naming it just `opts` — sure, I’ll do that. Personally I prefer to avoid shadowing, but I’ll go with your preference here. Thanks!